src/components/Records/Shared/RecordStatus.vue

```diff
@@ -9,18 +9,18 @@
     >
       <template #activator="{ on }">
         <v-avatar
-          v-if="Object.keys(recordType).includes(record.type)"
           size="80"
           :alt="getRecordStatus.title"
           v-on="on"
         >
-          <img
-            :src="('./' + recordType[record.type].icon)"
-            :alt="getRecordStatus.title"
-          >
+          <Icon
+            :item="record.type"
+            wrapper-class=""
+            :height="80"
```

Comment:
here if change the value to either lower like 60 or even higher like 100 the size of the icon remains the same.. 
based on the design the inner circle must be a little bit smaller than the dashed circle border. 

dev version
![image](https://user-images.githubusercontent.com/25683981/110482925-1861ed80-80e1-11eb-8a24-6826b3fa9654.png)

this branch version
![image](https://user-images.githubusercontent.com/25683981/110482854-041df080-80e1-11eb-807e-bdbc3bad6344.png)

Reply:
Im trying to fix it but there's so much css copied pasted all over the place that it's very very very hard to figure it out.